_includes/payment-transaction-states

```diff
@@ -0,0 +1,45 @@
+## Payment States And Transaction States
+
+Both payments and transactions find can be in several different states during
+the course of a purchase. You can find a short description of each state below.
+
+### Payments
+
+* `ready` means that the payment has been created successfully, and is now ready
+for further transaction operations, like `authorization`, `sale` or `abort`.
+
+* `failed` means that something went wrong during the payment process, and no
+further transactions can be created if the payment is in this state. Examples
+of possible failures are triggering of an antifraud pattern or if the payer
+reaches the maximum number of tries.
+
+* `aborted` means that the merchant has aborted the payment before the end user
+has fulfilled the payment process. This is done by calling the `PATCH` function
+for abort operations.
+
+* `pending` is the state of a payment when a transaction is in an `initialized`
+state. See more below. As long as a payment is `pending`, no further
+transactions can be done.
+
+
+### Transactions
+
+* `ìntialized` is the transaction state when something unexpected occured, and
+it is impossible to determine the exact status of the transaction. An example of
+this can be a network failure. No further actions can be done on a payment in
```

Comment:
Since we're talking about transactions and not payments in this section, their relationship needs to be explicitly explained. Would this be correct?

```suggestion
this can be a network failure. No further actions can be done on a payment with a transaction in
```